app/src/main/java/io/homeassistant/companion/android/sensors/CarSensorManager.kt

```diff
@@ -135,33 +191,34 @@ class CarSensorManager :
     }
 
     override fun hasSensor(context: Context): Boolean {
-        // TODO: show sensors for automotive (except odometer) once
-        //  we can ask for special automotive permissions in requiredPermissions
-        return Build.VERSION.SDK_INT >= Build.VERSION_CODES.O &&
-            !context.packageManager.hasSystemFeature(PackageManager.FEATURE_AUTOMOTIVE) &&
-            BuildConfig.FLAVOR == "full"
+        this.context = context.applicationContext
+
+        return Build.VERSION.SDK_INT >= Build.VERSION_CODES.O
```

Comment:
This will also make these sensors available on minimal builds when not running Automotive, which is undesirable as they will never get data. To summarise, they should be available if:

|     |`full` flavor|`minimal` flavor|
|-----|-----|-----|
|`hasSystemFeature(PackageManager.FEATURE_AUTOMOTIVE) == false` (phones, tablets, watches, ...)|yes (Android Auto)|no|
|`hasSystemFeature(PackageManager.FEATURE_AUTOMOTIVE) == true` (cars)|no (for now, lacking UI access)|yes (this PR)|

> Only available in Automotive Full flavor (Not in the Google Play version yet)

That's the wrong way around, full = Play Services

Reply:
Oh, I see. I thought the semantics were about the screens, not about play services. Fixed!